.pre-commit-config.yaml

```diff
@@ -99,6 +99,8 @@ repos:
           tests/test_integration/test_code_generation.py |
           tests/test_unittest/test_call_interface.py |
           tests/test_unittest/test_gtscript_frontend.py |
+          tests/test_unittest/test_compute_extents_pass.py |
+          tests/test_unittest/test_normalize_blocks_pass.py |
```

Comment:
@DropD The new tests trigger some `mypy` errors on these files, so I ignored them temporarily. These should be addressed before merging.
```
error: Unexpected keyword argument "start" for "AxisInterval"
error: Unexpected keyword argument "end" for "AxisInterval"
error: Unexpected keyword argument "level" for "AxisBound"
error: Unexpected keyword argument "offset" for "AxisBound"
```

Reply:
Yes, this has to do with the node metaclass, which is not recognized by mypy when used the way we do in gt4py. It is probably not worth going for a mypy solution there, since it only bothers us in tests and since the nodes will at some point in the future be based on a different framework which does not have that problem.